frontend/src/component/admin/network/NetworkOverview/NetworkOverview.tsx

```diff
@@ -24,30 +27,51 @@ const isRecent = (value: ResultValue) => {
 type ResultValue = [number, string];
 
 interface INetworkApp {
-    label?: string;
-    reqs: string;
+    label: string;
+    reqs: number;
     type: string;
 }
 
+const asNetworkAppData = (result: RequestsPerSecondSchemaDataResultItem) => {
+    const values = (result.values || []) as ResultValue[];
+    const data = values.filter(value => isRecent(value)) || [];
+    let reqs = 0;
+    if (data.length) {
+        reqs = parseFloat(data[data.length - 1][1]);
+    }
+    return {
+        label: unknownify(result.metric?.appName),
+        reqs: reqs,
+        type: unknownify(result.metric?.endpoint?.split('/')[2]),
+    };
+};
+
+const summingReqsByLabelAndType = (
+    acc: {
+        [group: string]: INetworkApp;
+    },
+    current: INetworkApp
+) => {
+    const groupBy = current.label + current.type;
+    acc[groupBy] = {
+        ...current,
+        reqs: current.reqs + (acc[groupBy]?.reqs ?? 0),
+    };
+    return acc;
+};
+
 const toGraphData = (metrics?: RequestsPerSecondSchema) => {
-    const results = metrics?.data?.result;
+    const results =
+        metrics?.data?.result?.filter(
+            result => unknownify(result.metric?.appName) !== 'unknown'
+        ) || [];
+    const aggregated = results
+        .map(asNetworkAppData)
+        .reduce(summingReqsByLabelAndType, {});
     return (
-        results
-            ?.map(result => {
-                const values = (result.values || []) as ResultValue[];
-                const data = values.filter(value => isRecent(value)) || [];
-                let reqs = 0;
-                if (data.length) {
-                    reqs = parseFloat(data[data.length - 1][1]);
-                }
-                return {
-                    label: unknownify(result.metric?.appName),
-                    reqs: reqs.toFixed(2),
-                    type: unknownify(result.metric?.endpoint?.split('/')[2]),
-                };
-            })
-            .filter(app => app.label !== 'unknown')
-            .filter(app => app.reqs !== '0.00') ?? []
+        Object.values(aggregated).filter(
+            app => app.reqs.toFixed(2) !== '0.00'
```

Comment:
Also feels like it could be done once instead of in the filter and then the graph below.